lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -113,16 +113,14 @@
                   ${form_html}
                 </div>
                 <p>
-                    ${_('After this purchase is complete,')}<br/><b>${order.user.username}</b>
-                    ${_('will be enrolled in this course.')}
+                    ${_('After this purchase is complete, {username_with_tags} will be enrolled in this course.').format(username_with_tags=u'<br/><b>{username}</b>'.format(username=order.user.username))}
```

Comment:
"username_with_tags" is too much detail for a translator. Why not just "username" or "name" so it's more clear to the translator